tools/taskcluster/build_targets.py

```diff
@@ -0,0 +1,88 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This script configures what APKs are built using the options passed in
+from the taskcluster {{ event.version }}.
+
+{{ event.version }} should be in the form: <tag name>:[all,release,debug]-<platform name>:[arm,arm64,x86]
+Some examples of {{ event.version }} and the resultant output from this script
+
+This is the default behaviour with no options. Only the Release build of each
+architecture for each supported platform is built:
+$ python build_targets.py 1.1.4a
+assembleWavevrArmRelease assembleNoapiArmRelease assembleNoapiArm64Release assembleNoapiX86Release assembleSvrArmRelease assembleSvrArm64Release assembleOculusvrArmRelease assembleOculusvrArm64Release assembleGooglevrArmRelease assembleGooglevrArm64Release
+
+Specifies only build the Arm64 version of the OculusVR platform:
```

Comment:
nit: should `Arm` be `ARM`?

Reply:
I matched the way it looks in Android Studio and for the gradle command line.